configure

```diff
@@ -24125,7 +24125,6 @@ else
 /* end confdefs.h.  */
 
       #include <openssl/crypto.h>
-      #include <openssl/fips.h>
```

Comment:
This file is not present for _e.g._ OpenSSL 1.0.2 installations, and is not necessary for FIPS detection.